webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading.js

```diff
@@ -345,6 +346,8 @@ export class SakaiRubricGrading extends RubricsElement {
       url: `/rubrics-service/rest/rubrics/${rubricId}?projection=inlineRubric`,
       headers: { "authorization": this.token }
     }).done(rubric => {
+	
+      if (!this.isPeerOrAuto) {
```

Comment:
What is Auto? Automatic grading?

Reply:
Better Self instead of Auto; is self-report